utils/ruleset-whitelist.csv

```diff
@@ -4,25 +4,44 @@ fbb2d56d33f886b53c314eac3446eed8bad7916660ad0709367936fe8335d987,0,1,112_Cafe.co
 5b650e2b6084539440a1b66b5b45f5825a2d61db7812c7521e1a1e0860c4b6a3,1,1,16163.com.xml
 e4f99c77dd7f504abcd6027a2ac0b7f916b29a634e855867ddc270ae531063c2,1,1,168qiquan.com.xml
 b5c96ae391a5b0971fb2438e3afd37ec7ce439e7dbb271276cf7bbeb0eb61450,1,1,16personalities.com.xml
-4d7ea51ecce8ef52d7f14fe3d2196e7647c347ae491a8c51ff84555426f4c05b,0,1,1phads.com.xml
 55b1ea082e1da07a13ef7bca833f779a23335658d345294a35e4944b90c5c136,0,1,1TW.org.xml
-0d24042a420f2069e4745384961064c2a016fb7ec4c094236ed565059581af4a,0,1,24-7_Customer.xml
+4d7ea51ecce8ef52d7f14fe3d2196e7647c347ae491a8c51ff84555426f4c05b,0,1,1phads.com.xml
```

Comment:
This is really minor but this looks like in the difference in the locales between the computer used to generate/clean the latest `ruleset-whitelist.csv` and the one you just used.

`ruleset-whitelist-cleanup.sh` should probably set the `LC_COLLATE` environment variable to prevent spurious diffs.

Reply:
Well, looks like it's already set to `C` via `LC_ALL` :thinking: 

https://github.com/EFForg/https-everywhere/blob/68425c9c903458f9874d6bb8942f428e34676f37/utils/ruleset-whitelist-cleanup.sh#L41

Reply:
Oh right, it's the first time it's run after https://github.com/EFForg/https-everywhere/pull/19025? Great job!

Reply:
There were some discrepancies with this file during the release process and that may have caused the issues with Chrome installs.